science/vis5d/Portfile

```
@@ -129,3 +128,8 @@ variant g95 description {compiles fortran interface for g95} { \
 #        configure.f77           ${prefix}/bin/gfortran-mp-4.3
 #        configure.env-delete    PTHREAD_LIBS=-lpthread
 #}
+
+notes-append {
+ImageMagick's "convert" is no longer included by default. \
```

Avoid double space after period:
```suggestion
ImageMagick's "convert" is no longer included by default.\
```